src/ts/state/test/reducer.spec.ts

```diff
@@ -613,12 +614,34 @@ describe("Action handlers", () => {
       expect(newState.decorations).toEqual(state.decorations);
     });
   });
-  describe("setDebug", () => {
-    it("should set the debug state", () => {
+  describe("setConfigValue", () => {
+    it("should set a config value", () => {
       const { state } = createInitialData();
       expect(
-        reducer(new Transaction(createDoc), state, setDebugState(true))
-      ).toEqual({ ...state, debug: true });
+        reducer(
+          new Transaction(createDoc),
+          state,
+          setConfigValue("debug", true)
+        )
+      ).toEqual({ ...state, config: { ...state.config, debug: true } });
+    });
+    it("should not accept incorrect config keys", () => {
+      const { state } = createInitialData();
+      reducer(
+        new Transaction(createDoc),
+        state,
+        // @ts-expect-error
```

Comment:
If you haven't come across `@ts-expect-error` before, you might have noticed – these tests don't assert anything! They're checking the typesafety of the `setConfigValue` function – [Typescript is expecting a type error there](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments), and fails if it doesn't see one. So the 'test' really happens at compile time.